release-notes.md

```diff
@@ -203,6 +212,79 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v24.0
+
+This release has been tracked through the [v24.0
+project](https://github.com/cloud-hypervisor/cloud-hypervisor/projects/28).
+
+### Bypass Mode for `virtio-iommu`
+
+`virtio-iommu` specification describes how a device can be attached by default
+to a bypass domain. This feature is particularly helpful for booting a VM with
+guest software which doesn't support `virtio-iommu` but still need to access
+the device. Now that Cloud Hypervisor supports this feature, it can boot a VM
+with Rust Hypervisor Firmware or OVMF even if the `virtio-block` device exposing
+the disk image is placed behind a virtual IOMMU.
+
+### Ensure Identifiers Uniqueness
+
+Multiple checks have been added to the code to prevent devices with identical
+identifiers to be created, and therefore avoid unexpected behaviors at boot or
+whenever a device was hot plugged into the VM.
+
+### Sparse Mmap support
+
+Adding sparse mmap support to both VFIO and vfio-user devices. This allows the
```

Comment:
Use the past tense: "Adding sparse mmap support to both VFIO and vfio-user devices" => "Sparse mmap support has been added to both ..."